.github/workflows/python.yml

```diff
@@ -37,8 +37,12 @@ jobs:
     name: AMD64 Ubuntu 18.04 Python 3
```

Comment:
Remove "18.04" from the name?

Reply:
Rather use the parameter from the matrix.